testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/custom/CustomThemeTest.java

```diff
@@ -61,26 +58,25 @@ public void configureTestRealm(RealmRepresentation testRealm) {
     protected LoginPage loginPage;
 
     @Page
-    protected AccountUpdateProfilePage profilePage;
+    protected AppPage appPage;
 
     // KEYCLOAK-3494
     @Test
-    public void changeProfile() throws Exception {
-        profilePage.open();
-        loginPage.login("test-user@localhost", "password");
+    public void changeProfile() {
+        UserRepresentation userRep = AccountHelper.getUserRepresentation(testRealm(), "test-user@localhost");
 
-        events.expectLogin().client("account").detail(Details.REDIRECT_URI, getAccountRedirectUrl()).assertEvent();
+        Assert.assertEquals("test-user@localhost", userRep.getEmail());
+        Assert.assertNull(userRep.firstAttribute("street"));
 
-        Assert.assertEquals("test-user@localhost", profilePage.getEmail());
-        Assert.assertEquals("", profilePage.getAttribute("street"));
+        userRep.singleAttribute("street", "Elm 1");
+        testRealm().users().get(userRep.getId()).update(userRep);
```

Comment:
After updating the `userRep` you have to re-fetch it from the DB for asserting the correct values.